app/controllers/api/storage.php

```diff
@@ -112,40 +584,49 @@
         }
 
         // Compression
-        $compressor = new GZIP();
         $data = $device->read($path);
-        $data = $compressor->compress($data);
-        $key = App::getEnv('_APP_OPENSSL_KEY_V1');
-        $iv = OpenSSL::randomPseudoBytes(OpenSSL::cipherIVLength(OpenSSL::CIPHER_AES_128_GCM));
-        $data = OpenSSL::encrypt($data, OpenSSL::CIPHER_AES_128_GCM, $key, 0, $iv, $tag);
+        if($size <= APP_LIMIT_COMPRESSION) {
```

Comment:
Lets use strict comparison `===`

Reply:
Don't think there is something like <==?

Reply:
Same, I don't think this operator exists.